revdb/revdb.gyp

```diff
@@ -278,12 +292,22 @@
 							'../thirdparty/libiodbc/libiodbc.gyp:libiodbc',
 						],
 					},
+					'OS == "win"',
```

Comment:
I'm not sure this is still quite right... Gyp conditions need to be laid out as follows:
```
'conditions':
[
  [
    'condition expr',
    {
      consquent dict
    },
    {
      alternate dict
    },
  ],
  [
    '<next condition expr>',
    ...
  ],
  ...
],
```

Reply:
Arghh! I'll get this right at some point.